app/views/articles/tag_index.html.erb

```diff
@@ -33,6 +33,7 @@
       data-params="<%= params.to_json(only: %i[tag username q]) %>" data-which=""
       data-algolia-tag="<%= @tag %>"
       data-feed="<%= params[:timeframe] || "base-feed" %>"
+      data-requires-approval="<%= @tag_model.requires_approval || false %>"
```

Comment:
The `|| false` is probably superfluous here, eh? Since `requires_approval` will output false if not true?

Reply:
Correct, and fixed!